sandbox/src/personalizationAnalyticsClientSideHelper.js

```diff
@@ -64,14 +65,17 @@ const sendEvent = ({
   return window[instanceName]("sendEvent", {
     renderDecisions,
     decisionScopes,
+    personalization,
     xdm
   });
 };
 
 export const getFormBasedOffer = () => {
   sendEvent({
     eventType: "form-based-offer",
```

Comment:
Same here, let's leave everything as is.

Reply:
Reverted